libraries/networking/src/ResourceCache.cpp

```diff
@@ -402,7 +402,7 @@ void Resource::handleReplyFinished() {
                 const int MAX_ATTEMPTS = 8;
                 const int BASE_DELAY_MS = 1000;
                 if (++_attempts < MAX_ATTEMPTS) {
-                    QTimer::singleShot(BASE_DELAY_MS * (int)pow(2.0, _attempts), this, SLOT(attemptRequest()));
+                    QTimer::singleShot(BASE_DELAY_MS * (int)pow(2.0, _attempts), this, &Resource::makeRequest);
```

Comment:
Hmm, `GeometryCache` no longer subclasses `ResourceCache` so that shouldn't be a problem. I'm not sure if `attemptRequest` was ever virtual, but it isn't now. I do agree requestCompleted could be moved up if that's what you mean. In fact I think everything in the if could.